### ISSUE TITLE
hot-key support

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#1034]
Make it so people can use hot-keys (probably require them to be control characters initially).

This is a placeholder. I'll fill in more details "soon".

**Comment by ctm** [comment]
FTR, [Here's how PokerStars does it](https://www.stars-support.net/articles/keyboard-shortcuts/11044/?br=pokerstars).

**Comment by ctm** [comment]
Here's a [useful page](https://javascript.info/bubbling-and-capturing) on event bubbling.

**Comment by ctm** [comment]
I didn't screenshot the PokerStars page and now it might not lead somewhere useful. However, the BARGE list just had a post that contained:

> [...] web based free poker site. It’s incredibly well done and with single keys for actions the game flies by. Check it out

[https://www.pokernow.club/](https://www.pokernow.club/)

I'm labeling this temporarily `high priority`, but will knock it back down to normal priority after I figure out how to handle chat and hot keys.

**Comment by ctm** [comment]
FWIW, I added a keyup and keydown listener to the document just to verify that I can capture the keys easily (and I can). I'm not doing anything with the keypresses yet, but I did make it so that the rotate icon flips when the shift key is down.

**Comment by ctm** [comment]
I actually added an undocumented `!` command that uses the listener, so really the only thing holding me back is just how do I allow players to choose mappings. The easiest would be to simply have one set of mappings and let players use them or not, but that's likely to lead to requests for more changes. As such, I probably won't do anything with this for a little while longer, but at least I'm no longer blocked by not wanting to add more UIKit code, since Claude and I removed UIKit.
